backend/src/scripts/migrations/4.30/addVertexCountsToMeshBSONs.js

```diff
@@ -0,0 +1,88 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { v5Path } = require('../../../interop');
+
+const { UUIDToString, stringToUUID } = require(`${v5Path}/utils/helper/uuids`);
+const { getTeamspaceList, getCollectionsEndsWith } = require('../../utils');
+
+const { find, updateOne } = require(`${v5Path}/handler/db`);
+const { logger } = require(`${v5Path}/utils/logger`);
+
+const processModel = async (teamspace, model) => {
+	const stashDB = `${model}.stash.3drepo`;
```

Comment:
can we add a test here to see if there's any meshes in the `.scene` collection that doesn't have vertex_counts? 

it's quite likely we will have to rerun the script, we'd want to early return and don't do all the ops if we've done it already.